README.md

```diff
@@ -1,8 +1,27 @@
 # Maven Plugin for building Galasa
 
-## Project status
-[![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasa-maven-plugin&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasa-maven-plugin)
-[![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasa-maven-plugin&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasa-maven-plugin)
-[![Coverage](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasa-maven-plugin&metric=coverage)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasa-maven-plugin)
+This respository is an extension to Maven and contains code that is used to build OSGI bundles for Galasa. The OSGI bundle repositories contain all the project information, configuration details and dependencies that are needed for building and running Galasa projects.
```

Comment:
change "contains code" to "includes goals".  Also the maven plugin includes a goal to build the test catalogs.
Think "extension to Maven2 is better as the build "Maven Plugin".